artifactory/commands/golang/go.go

```
@@ -106,9 +109,26 @@ func (gc *GoCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	// Extract vcs fallback from the args.
+	gc.goArg, err = gc.extractVcsFallbackFromArgs()
```

Let's not ignore the error returned here.